src/panels/config/integrations/integration-panels/zwave_js/zwave_js-node-config.ts

```diff
@@ -369,6 +416,18 @@ class ZWaveJSNodeConfig extends SubscribeMixin(LitElement) {
     return [
       haStyle,
       css`
+        .accepted {
+          color: green;
+        }
+
+        .queued {
+          color: #fca503;
```

Comment:
Use the variables we have for this: 
```
      --error-color: #db4437;
      --warning-color: #FF9800;
      --success-color: #0f9d58;
      --info-color: #4285f4;
```

Reply:
Aha, I wondered if something like that existed. Is there a list of variables somewhere?